private-templates-service/client/src/Welcome.tsx

```diff
@@ -1,15 +1,22 @@
-import React from 'react';
-import {
-  Button,
-  Jumbotron } from 'reactstrap';
+import React from "react";
+import { Button, Jumbotron } from "reactstrap";
 
-function WelcomeContent(props) {
+interface Props {
+  isAuthenticated: boolean;
+  user: any;
+  authButtonMethod: any;
+}
+
+function WelcomeContent(props: any) {
```

Comment:
props: WelcomeContentProps

Reply:
or just WelcomeContent(user: User) since I think that's all you are pulling from props

Also arrow functions